plugins/UM3NetworkPrinting/ClusterControlItem.qml

```diff
@@ -25,217 +30,663 @@ Component
+                                }
+                            }
+
+                            Rectangle
+                            {
+                                id: jobSpacer
+                                color: UM.Theme.getColor("viewport_background")
+                                height: 2
+                                anchors
+                                {
+                                    left: parent.left
+                                    right: parent.right
+                                    margins: UM.Theme.getSize("default_margin").width
+                                    top: extrudersInfo.bottom
+                                    topMargin: 2 * UM.Theme.getSize("default_margin").height
+                                }
+                            }
+
+                            Item
+                            {
+                                id: jobInfo
+                                property var showJobInfo: modelData.activePrintJob != null && modelData.activePrintJob.state != "queued"
+
+                                anchors.top: jobSpacer.bottom
+                                anchors.topMargin: 2 * UM.Theme.getSize("default_margin").height
+                                anchors.left: parent.left
+                                anchors.right: parent.right
+                                anchors.margins: UM.Theme.getSize("default_margin").width
+                                anchors.leftMargin: 2 * UM.Theme.getSize("default_margin").width
+                                height: showJobInfo ? childrenRect.height + 2 * UM.Theme.getSize("default_margin").height: 0
+                                visible: showJobInfo
+                                Label
+                                {
+                                    id: printJobName
+                                    text: modelData.activePrintJob != null ? modelData.activePrintJob.name : ""
+                                    font: UM.Theme.getFont("default_bold")
+                                    width: parent.width
+                                    elide: Text.ElideRight
+                                }
+                                Label
+                                {
+                                    id: ownerName
+                                    anchors.top: printJobName.bottom
+                                    text: modelData.activePrintJob != null ? modelData.activePrintJob.owner : ""
+                                    font: UM.Theme.getFont("default")
+                                    opacity: 0.6
+                                    width: parent.width
+                                    elide: Text.ElideRight
+                                }
+
+                                function switchPopupState()
+                                {
+                                    popup.visible ? popup.close() : popup.open()
```

Comment:
Just use a normal `if()` here and leave the `?:` for expressions.

Reply:
This was copied from the other cura code.